### ISSUE TITLE
Consider making it possible to choose fonts

### ISSUE THREAD

**Issue by toolness** [toolness/accessible-color-matrix#12]
After reading the **Anatomy of a Typeface** lesson in [Better Web Typography](https://betterwebtype.com/), I noticed that the author talks about how different fonts have different levels of contrast:

> <img src="https://gallery.mailchimp.com/d2aaa3155cfda7e76c73817a0/images/4e17c313-a77e-45c5-809b-32a9f46de4ac.jpg">
>
> Figure 19: Contrast is also a major contributing factor to how “heavy” a typeface looks like. Merriweather is a low-contrast typeface, while Bodoni is a high-contrast one. Take a closer look at each letter strokes. The difference is obvious.

I wonder if it'd be useful to take this into account somehow in the palette builder.

For instance, in the "type specimens" in the color matrix, might it be useful to allow the user to choose a typeface to use, so that they can see what it looks like in given color combinations.

Thoughts @ericronne?

**Comment by ericronne** [comment]
Yeah, that could be useful to designers, i'm sure. Almost edges it toward style guide territory. We could mark that for future exploration.
